src/elementPlugins/dropdownChoice/src/ApplicationView.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from 'react'
+import { Dropdown, Header, Message } from 'semantic-ui-react'
+import { ApplicationViewProps } from '../../types'
+
+const ApplicationView: React.FC<ApplicationViewProps> = ({
+  templateElement,
+  onUpdate,
+  initialValue,
+  isEditable,
+  currentResponse,
+  validationState,
+  onSave,
+}) => {
+  const {
+    label,
+    description,
+    placeholder,
+    options,
+    default: defaultIndex,
+  } = templateElement?.parameters
+
+  const [value, setValue] = useState(initialValue ? initialValue?.text : options[defaultIndex])
```

Comment:
what would be the difference between condition operator and `initialValue?.text || options[defaultIndex]`

Reply:
Yeah, I guess that works and is a bit more concise. Thanks.